listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -564,7 +570,14 @@ export default class RecentListens extends React.Component<
                 <PinnedRecordingCard
                   userName={user.name}
                   pinnedRecording={userPinnedRecording}
+                  className={
+                    this.isCurrentListen(getListenablePin(userPinnedRecording))
```

Comment:
I am not sure this will work as expected. Should this effect also take place if a listen of the same recording which is pinned is being played ? In that case, this won't work. I am fine either way but wanted to confirm the intent.

Reply:
Yes it works as intended, if BrainzPlayer is playing something only one card component should be highlighted at a time whether it's a listen or pin.